aliyun/src/test/java/org/apache/iceberg/aliyun/oss/TestOSSOutputFile.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.aliyun.oss;
+
+import com.aliyun.oss.OSS;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.Random;
+import java.util.UUID;
+import java.util.concurrent.ThreadLocalRandom;
+import org.apache.commons.io.IOUtils;
+import org.apache.iceberg.AssertHelpers;
+import org.apache.iceberg.aliyun.AliyunProperties;
+import org.apache.iceberg.exceptions.AlreadyExistsException;
+import org.apache.iceberg.io.InputFile;
+import org.apache.iceberg.io.OutputFile;
+import org.apache.iceberg.relocated.com.google.common.io.ByteStreams;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class TestOSSOutputFile extends AliyunOSSTestBase {
+
+  private final OSS ossClient = ossClient().get();
+  private final Random random = ThreadLocalRandom.current();
+  private final AliyunProperties aliyunProperties = new AliyunProperties();
+
+  @Test
+  public void testWriteFile() throws IOException {
+    OSSURI uri = randomURI();
+
+    int dataSize = 1024 * 1024 * 10;
+    byte[] data = randomData(dataSize);
+
+    OutputFile out = OSSOutputFile.fromLocation(ossClient, uri.location(), aliyunProperties);
+    try (OutputStream os = out.createOrOverwrite()) {
+      IOUtils.write(data, os);
```

Comment:
Same comment as this one: https://github.com/apache/iceberg/pull/3404#discussion_r739861968